### DISCUSSION TITLE
Docker Compose File in Yaml Format Is Supported?

### DISCUSSION THREAD

**Discussion by kaenova** [devcontainers/community#102] [Q&A]
First of all, thanks to all developers and contributors of devcontainer!
Devcontainer open an easy way to manage multi-service development for me!

It's just happened to me when I follow a guide from [Microsoft's YouTube channels on configuring multiple devcontainer](https://youtu.be/bVmczgfeR5Y?si=EYcHtmDpf0D7RGT5) in one project, especially when opening a container using command in the vscode `Dev Container: Reopen in Container`, I am configuring the `docker-compose` file wrong. Instead of `docker-compose.yml`, I named it `docker-compose.yaml` and the the vscode command doesn't pick up the configuration like it should be. Is it that the yaml file of the `docker-compose` is not supported? Thanks!

**Comment by samruddhikhandale** [comment]
Hi 👋 

Thanks for your kind words. Yes, I believe we currently support only the `yml` format of `docker-compose` file. Hence, the picker would fail to identify the `.yaml` extension.
